roles/KubevirtTemplateValidator/defaults/main.yml

```diff
@@ -2,5 +2,4 @@
 # defaults file for KubevirtTemplateValidator
 wait: true
 validator_image: kubevirt-template-validator
-# this comes from the CR, we must diverge from validator_$SOMETHING standard naming
-version: "{{ validator_tag |default('v0.6.2') }}"
+validator_version: "{{ version | default(validator_tag, true) }}"
```

Comment:
Why not use the same like in templates?

version: "{{ validator_tag }}"

It is already a defaults file and should behave like we want. Then just use {{ version }} in the tasks.

Reply:
It's just me being inexperienced in ansible. Done.